ddtrace/contrib/aws_lambda/patch.py

```diff
@@ -121,19 +121,27 @@ def _get_handler_and_module():
     path = os.environ.get("DD_LAMBDA_HANDLER", None)
     _datadog_instrumentation = DatadogInstrumentation()
 
+    handler, handler_module = None, None
+
```

Comment:
nit: `handler`  and `handler_module` should never be None. This will cause `unwrap(handler, wrapper)` to fail. 
```suggestion
```

Reply:
Would it make sense to have two returns here again?